tests/test_transforms/test_inference/test_transform.py

```diff
@@ -206,7 +206,10 @@ def _test_transform_train_subset_segments(self, ts, transform, segments):
                 ),
                 "ts_to_resample",
             ),
-            (TimeSeriesImputerTransform(in_column="target"), "ts_to_fill"),
+            (TimeSeriesImputerTransform(in_column="target", strategy="constant"), "ts_to_fill"),
```

Comment:
Why there aren't `running_mean` and `forward_fill`? How have you chosen these one?

Reply:
Fixed